packages/utils/src/is.js

```diff
@@ -37,6 +37,8 @@ export const is = {
 	file: (o) => o instanceof File,
 	numeric,
 	numericZero,
+	/** @type {(o: any) => o is import('create-emotion').ObjectInterpolation} */
+	objectInterpolation: (o) => _.isPlainObject(o),
```

Comment:
This is kind of a hack? What do you think @sirreal?

Reply:
Yeah… I assume this was to satisfy the compiler? I'd rather use a careful assertion than export a new, relatively weak "is" funciton 🙂

Reply:
I'd rather that too, but I could not figure out what assertion to use 🤔 `ObjectInterpolation` is basically a special `Record` with specific keys, right?

```
{
  [K in keyof CSSProperties]:
    | CSSProperties[K]
    | Array<Extract<CSSProperties[K], string>>
}
```

And because `Interpolation` also includes `string` `number` and `null` `undefined` you have to do a bunch of assertions to get here. Any ideas?

Reply:
I think I've got it... we can make this safe by only allowing `TemplateStringsArray | Interpolation` to be passed in.